### ISSUE TITLE
Fail fast on duplicated parameterized entries

### ISSUE THREAD

**Issue by kornicameister** [typeddjango/pytest-mypy-plugins#33]
Ref: https://github.com/kornicameister/loguru-mypy/pull/31#discussion_r470841984

---

Idea here is to check if `parametrized` contains a duplicated dictionaries and if so fail fast the test.
Problem though is that `pytest` behavior on parameterized is different...basically `pytest` does not seem to care if you have duplicates. 

Thing to think about for a minute, I guess :)

**Comment by sobolevn** [comment]
Yes, let's do that! Each test runs for a while. And it might be helpful to find duplicates and reduce the overall time.

**Comment by kornicameister** [comment]
I can't promise when :( if someone else is willing to jump in, go ahead ;) I will happily review.
